tests/configuration.py

```diff
@@ -0,0 +1,105 @@
+import logging
+
+from django.conf import settings
+from django.test import TestCase
+from django.test.utils import override_settings
+
+from django_emarsys.test_helper import MockLoggingHandler
+from django_emarsys import util
+
+
+class ModelTestCase(TestCase):
+    @override_settings()
+    def test_configuration_present(self):
+        logger = logging.getLogger(util.__file__)
+        handler = MockLoggingHandler(level='WARNING')
+        logger.addHandler(handler)
+
+        # remove all EMARSYS_* settings
+        for symbol in dir(settings):
+            if symbol.startswith('EMARSYS_'):
+                delattr(settings, symbol)
+
+        util.validate_configuration()
+
+        expected_messages = set([
+            'no EMARSYS_ACCOUNT setting set',
+            'no EMARSYS_PASSWORD setting set',
+            'no EMARSYS_BASE_URI setting set',
+            'no EMARSYS_EVENTS setting set',
+        ])
+        actual_messages = [message.getMessage()
+                           for message in handler.messages]
+
+        for message in actual_messages:
+            if message not in expected_messages:
+                raise self.failureException("got unexpected log message '{}'"
+                                            .format(message))
+
+            expected_messages.remove(message)
+
+        if expected_messages:
+            raise self.failureException(
+                "expected log message(s) missing: {}"
+                .format(list(expected_messages)))
```

Comment:
assertEqual does all that for you by calling https://docs.python.org/3.5/library/unittest.html#unittest.TestCase.assertSetEqual . (Django includes unittest2, which is unittest from Python 3)